cfme/tests/networks/nuage/test_inventory.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+from cfme.networks.provider.nuage import NuageProvider
+
+pytestmark = [
+    pytest.mark.provider([NuageProvider], scope='module')
+]
+
+
+def test_tenant_details(setup_provider_modscope, provider, with_nuage_sandbox_modscope):
+    sandbox = with_nuage_sandbox_modscope
+    tenant_name = sandbox['enterprise'].name
+    tenant = provider.cloud_tenants.get_by_name(tenant_name)
+
+    subnets = []
+    [subnets.extend(z.subnets) for d in sandbox['enterprise'].domains for z in d.zones]
+    l2_groups = sum([len(d.policy_groups) for d in sandbox['enterprise'].l2_domains])
+    groups = sum([len(d.policy_groups) for d in sandbox['enterprise'].domains])
+    l2_vports = sum([len(d.vports.get()) for d in sandbox['enterprise'].l2_domains])
+    vports = sum([len(d.vports.get()) for d in sandbox['enterprise'].domains])
+
+    expected_stats = {
+        'num_cloud_subnets': len(subnets) + len(sandbox['enterprise'].l2_domains.get()),
+        'num_network_routers': len(sandbox['enterprise'].domains),
+        'num_security_groups': l2_groups + groups,
+        'num_network_ports': l2_vports + vports
+    }
```

Comment:
So sorry to mislead you earlier @aljazkosir but we know all those values upfront - because we know exactly what we created in the sandbox. Can you please just hard-code them here?

```
expected_stats = {
	'num_cloud_subnets': 1,
	'num_network_routers': 1,
	'num_security_groups': 2,
	'num_network_ports': 4
}
```

The idea is that we grab only those variables from sandbox that we don't know upfront (e.g. tenant name - because tenant name is randomized upon creation).